src/hooks/useIsHostIn/useIsHostIn.tsx

```diff
@@ -66,11 +66,11 @@ export default function useIsHostIn() {
     if (theRoom !== null && typeof theRoom.participants !== 'undefined') {
```

Comment:
Let's use the same constant in every place - so we can change the constant value and everything will continue to work

Reply:
I can make it a constant.
But that's not how you write in  React,